### PR TITLE
Scalafmt add

### PR DESCRIPTION
**Pull request by sd1ver**
Scalafmt is a code formatter for Scala. It is integrated with IDEA

### CODE REVIEW

File: scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+includeCurlyBraceInSelectChains = false
+danglingParentheses = true
+spaces {
+  inImportCurlyBraces = true
+}
+optIn.annotationNewlines = true
+verticalMultiline.atDefnSite = true
+verticalMultiline.newlineAfterOpenParen = true
+verticalMultiline.arityThreshold = 100
+rewrite.rules = [SortImports]
```
Comment (Primetalk) [inline scalafmt.conf]: no line end

File: scalafmt.conf [hunk 2]
```diff
@@ -0,0 +1,16 @@
+version = "2.3.2"
```
Comment (Primetalk) [inline scalafmt.conf]: Filename should be .scalafmt.conf, AFAIK.

File: scalafmt.conf (hunk 1 above)
Comment (sd1ver) [reply scalafmt.conf]: added

File: scalafmt.conf (hunk 2 above)
Comment (sd1ver) [reply scalafmt.conf]: Renamed

**Pull request merged**